src/edu/csus/ecs/pc2/ui/MultipleDataSetPane.java

```diff
@@ -751,4 +788,26 @@ public void actionPerformed(ActionEvent e) {
         }
         return chckbxStopOnFirstFailedTestCase;
     }
+
+    private JCheckBox getLoadSamplesFirstCheckbox() {
+        if (loadSamplesFirstCheckbox == null) {
+            loadSamplesFirstCheckbox = new JCheckBox("Load Sample Data");
```

Comment:
If I understand correctly the intent of this PR, it is to provide support for also loading SAMPLE data when the "judge's data" (a.k.a. "secret" data) is loaded [*].  Assuming my understanding is correct, I think it would be much clearer to have the label on this checkbox be "Include Sample Data" rather than "Load Sample Data".   The latter phrasing has the potential to be confusing:  it's not clear whether it means "load _only_ the Sample Data" or "when you load the (secret) data, _also_ load the sample data".   I think (assuming I'm right that the intent of the PR is to support _also_ loading Sample Data) that a label like "Include Sample Data" -- or maybe even better, "Include Sample Data on Loading" -- would be clearer.

[*] Having said the above, it's possible I have a major misunderstanding of the intent of this PR; I'm having a LOT of trouble getting it to work the way I would expect (and so maybe my expectations are wrong).  I will follow up on this issue with a separate comment.

Reply:
Whoa... sudden revelation:  this PR (according to it's title) is about _Adding a pane to submit judges solutions_.  Now I'm questioning why there is stuff in here related to loading Sample (vs. Secret) data... it seems like that is a serious overlap of Issues...

Reply:
I consider removing all changes having to do with loading sample vs. secret data an approval showstopper.